src/partition.rs

```diff
@@ -662,7 +662,8 @@ pub const GLOBALMV_CTX_MASK: usize =
 pub const REFMV_CTX_MASK: usize = ((1 << (8 - REFMV_OFFSET)) - 1);
 
 pub static RAV1E_PARTITION_TYPES: &'static [PartitionType] =
-  &[PartitionType::PARTITION_NONE, PartitionType::PARTITION_SPLIT];
+  &[PartitionType::PARTITION_NONE, /*PartitionType::PARTITION_HORZ,
+    PartitionType::PARTITION_VERT,*/ PartitionType::PARTITION_SPLIT];
```

Comment:
Why?

Reply:
As 'Prep' means, this patch has not enabled actual use of horz and vert yet!